crates/wash-cli/src/up/mod.rs

```diff
@@ -233,6 +233,9 @@ pub struct WasmcloudOpts {
     )]
     pub enable_structured_logging: bool,
 
+    #[clap(short = 'l', long = "label")]
```

Comment:
NIT: comment explaining the option would be good here... also what do you think about an alias `labels`?